.github/workflows/chatbot.yaml

```diff
@@ -58,6 +58,10 @@ jobs:
         working-directory: ./recipes/natural_language_processing/${{ env.IMAGE_NAME }}
         run: make install
 
+      - name: Download model
+        working-directory: ./model_servers/${{ matrix.image_name }}/
+        run: make download-model-mistral
```

Comment:
I think this is blowing up in PR?

Reply:
Yeah, but I also realized that it requires both chrome and the chrome driver with a paired version, Ive been reworking this.